setup.sh

```diff
@@ -298,13 +298,23 @@ function synchronize_files()
   setup_config_file
   ASSERT_IF_NOT_EQ_ZERO "Config file failed" "$?"
 
-  if [ -f "$HOME/.bashrc" ]; then
-      # Add to bashrc
-      echo "# $app_name" >> "$HOME/.bashrc"
-      echo "source $libdir/$BASH_AUTOCOMPLETE.sh" >> "$HOME/.bashrc"
-      update_path
+  if [ -f "$HOME/.bashrc" ] || [ -h "$HOME/.bashrc" ]; then
```

Comment:
Why not use `[[ -f "..." || -h "..." ]]`?

Reply:
This file was originally using `if [ -f "$HOME/.bashrc" ]`  and the codestyles file didnt mention anything about using [] or [[]] so I tried to stay consistent by keeping []. To be honest I prefer [[]] haha should I change it?

Reply:
I changed it to [[ -f "..." || -h "..." ]]